src/toil_rnaseq_sc/rnaseq_sc_cgl_pipeline.py

```diff
@@ -79,19 +81,20 @@ def run_single_cell(job, config, sample):
     with open(os.path.join(work_dir, "config.json"), 'w') as config_file:
         config_file.write(build_patcherlab_config(config))
     # Get Kallisto index
-    download_url(job, url=config.kallisto_index, name='kallisto_index.idx', work_dir=work_dir)
+    download_url(url=config.kallisto_index, name='kallisto_index.idx', work_dir=work_dir)
     # Get input files
     input_location = os.path.join(work_dir, "fastq_input")
     os.mkdir(input_location)
     uuid, urls = sample
+    config.uuid = uuid
```

Comment:
Add this line: `config = argparse.Namespace(**vars(config))` to make a unique config for this sample.